docs/learning/quantity/index.md

```diff
@@ -305,3 +307,51 @@ If a Format uses a spacer that conflicts with one of the operators above, additi
 ```
 
 </details>
+
+## Persistence
+
+We expose APIs and interfaces to support persistence of formats. Different from [KindOfQuantity](../../bis/ec/kindofquantity.md), which enables persistence of formats at the schema level, this section covers persistence at the application level.
+
+### PresentationFormatSet
+
+[PresentationFormatSet]($quantity) defines properties necessary to support persistence of a set of formats.
+
+Formats defined in a PFS needs to be mapped to the a valid [ECName](../../bis/ec/ec-name.md) for a [KindOfQuantity](../../bis/ec/kindofquantity.md). During an application's runtime, the format associated to a KoQ within a PFS would take precedence and be used over the default presentation formats of that KoQ.
+
+> The naming convention for a valid PFS format <full-schema-name>:<koq-name>
+.
+<details>
+<summary>Example of a Presentation Format Set as JSON:</summary>
+
+```json
+{
+  "id": "metric",
+  "label": "Metric",
+  "formats": {
+    "AecUnits.LENGTH": {
+      "composite": {
+        "includeZero": true,
+        "spacer": "",
+        "units": [{ "label": "m", "name": "Units.M" }]
+      },
+      "formatTraits": ["keepSingleZero", "showUnitLabel"],
+      "precision": 4,
+      "type": "Decimal",
+      "decimalSeparator": "."
+    },
+    "AecUnits.Angle": {
+      "id": "Angle",
```

Comment:
what us the purpose of this id property?

Reply:
It represents a unique identifier for the format, separate from the identifier of a KindOfQuantity. This id shouldn't be user facing, and is relevant when working with formats retrieved from schemas, for example when overriding  the property of the format in session/runtime

Reply:
in retrospect, I should not have named the id `angle` as an example, it should be more descriptive/ tells you it comes from a schema perhaps